app/models/user.rb

```diff
@@ -0,0 +1,25 @@
+class User < ApplicationRecord
+  validates :name, presence: true
+  validates :photo, presence: true
+  validates :post_counter, presence: false
+
+  has_many :comments
+  has_many :likes
+  has_many :posts
```

Comment:
- [ ] The column on the comments, likes and posts tables which helps to implement the `has-many` relationship between them( the models) and `User`, is `author_id` as you can see in the attached ERD diagram below 👇 

I noticed that you made changes to your schema to be so that the foreign keys on the `comments`, `posts`, and `likes` tables are `user_id` instead of `author_id`. I highly recommend keeping your database tables and columns consistent with the ERD diagram below.

**HINT:** You can set up your model relationships this way 👇 
```ruby
`# app/models/user.rb`
has_many :posts, foreign_key: 'author_id'
has_many :comments, foreign_key: 'author_id'
has_many :likes, foreign_key: 'author_id'
```

Then in the `comment`, `like`, and `post` model, do this 👇 
```ruby
belongs_to :author, class_name: 'User'
```

![image](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png?raw=true)